src/main/java/org/aksw/simba/lemming/colour/ColourPaletteWrapper.java

```diff
@@ -0,0 +1,56 @@
+package org.aksw.simba.lemming.colour;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.io.Serializable;
+import java.util.BitSet;
+
+/**
+ * Wrapper Class to enable the persistence of the ColourPalette class
+ * @author Ana
+ *
+ */
+public class ColourPaletteWrapper implements Serializable{
+	
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = 3744241658289486040L;
+	protected Map<String, BitSet> uriColourMap = new HashMap<String, BitSet>();
+    protected int highestColourId = 0;
```

Comment:
Do we really need to keep track of this id? Isn't it automatically define by the size of the uriColourMap?